src/dotnet/commands/dotnet-restore/Program.cs

```diff
@@ -24,16 +24,12 @@ public static int Run(string[] args)
                 Description = "Restores dependencies listed in project.json"
             };
 
-            // Parse --quiet, because we have to handle that specially since NuGet3 has a different
-            // "--verbosity" switch that goes BEFORE the command
```

Comment:
Yes, passing --verbosity|-v does work. (In the build log from CI you can see some commands passing --verbosity and getting full output.)

Frankly, I'm not sure what that comment really meant.